CONTRIBUTING.md

```diff
@@ -19,7 +19,7 @@ Please ensure that your pull request adheres to the following guidelines:
 - Maintainer confirmation is also required in case repository has more than one beginner-friendly-like label (e.g. `low-hanging-fruit` and `up-for-grabs`).
 - The contributed repository must be maintained, have an active community to be able to help newcomers and have issues with an appropriate label.
 - Links must be pointing directly to the repository, no tracking links allowed. This list is not for advertising purposes.
-
+shall we have a french version.
```

Comment:
Hello @zhongch , I assume this is a new feature request. I suggest you to raise an issue first instead of PR. Thanks

Reply:
### thanks for the reply.